packages/openapi-v3/src/__tests__/unit/enhancers/fixtures/security.spec.extension.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {bind} from '@loopback/core';
+import debugModule from 'debug';
+import {inspect} from 'util';
+import {
+  defaultMergeFn,
+  ReferenceObject,
+  SecuritySchemeObject,
+} from '../../../..';
+import {asSpecEnhancer, OASEnhancer} from '../../../../enhancers/types';
+import {OpenApiSpec} from '../../../../types';
+const debug = debugModule('loopback:openapi:spec-enhancer');
+
+export type SecuritySchemeObjects = {
+  [securityScheme: string]: SecuritySchemeObject | ReferenceObject;
+};
+
+export const SECURITY_SCHEME_SPEC: SecuritySchemeObjects = {
+  bearerAuth: {
+    type: 'http',
+    scheme: 'bearer',
+    bearerFormat: 'JWT',
+  },
+};
+
+/**
+ * A spec enhancer to add bearer token OpenAPI security entry to
+ * `spec.component.securitySchemes`
+ */
+@bind(asSpecEnhancer)
+export class SecuritySpecEnhancer implements OASEnhancer {
+  name = 'security';
+
+  modifySpec(spec: OpenApiSpec): OpenApiSpec {
```

Comment:
Security stuff can be added at the top level, or for operation level. Does this method take this into account?

Reply:
e.g. described here : https://loopback.io/doc/en/lb4/Authentication-Tutorial.html#specifying-the-security-settings-in-the-openapi-specification

Reply:
The operation level spec will be addressed in a new story, see all the discussion in https://github.com/strongloop/loopback-next/pull/4258#discussion_r358465354, me and @bajtos tend to introduce `@security` to provide the spec. So the extension in this PR only focus on merging the security scheme into components.